main/arkgo-gui.go

```diff
@@ -543,9 +543,36 @@ func main() {
 	viper.AddConfigPath("settings") // path to look for the config file in
 	viper.AddConfigPath(".")        // optionally look for config in the working directory
 	err := viper.ReadInConfig()     // Find and read the config file
-	if err != nil {                 // Handle errors reading the config file
-		panic(fmt.Errorf("Fatal error config file: %s \n", err))
+
+  if err != nil {
+		logger.Println("No productive config found - loading sample")
 	}
+
+  //Load defaults
```

Comment:
I suggest to put this logic - into a void function... func initSettings() {..} and call it from main.... it would be easier to maintain...